new/rfc-for-hugs.md

```diff
@@ -0,0 +1,50 @@
+---
+RFC: unassigned
+Author: Aaron Kalin and JJ Asghar
+Status: Draft
+Type: Informational, Process
+---
+
+# Hugging in the Chef Community
+
+In order to bridge the cultural gap between the US and Worldwide community the concept of hugging should be explained.
+
+## Motivation
+
+    As a Chef Community Participant,
+    I want to express positive emotion to or for another Chef Community Participant,
+    so that we can express our friendship physically.
+
+## Specification
+
+When approaching another Chef Community Member that you wish to engage in a hug. In order to comply with the already
+existing Chef Community Guidelines and Code of Conduct at various community events. You must seek permission from the
```

Comment:
The "In order" sentence does not make sense, it ends prematurely, or possibly should end in a comma before "You must".